# Avoid a forced copy of the 'pymolcas' script in a local path
# of the user doing the build
# 'make install' will install it using the proper prefix
# by A. Lozano (ULiege)
--- a/sbin/install_pymolcas.sh	2018-09-02 09:35:13.000000000 +0200
+++ b/sbin/install_pymolcas.sh 	2019-03-11 16:53:29.224546598 +0100
@@ -116,8 +116,8 @@
   echo "Is this OK? [Y/n] (will assume \"Yes\" in 25 seconds)"
   while true ; do
     if [ "$INTERACTIVE" = "0" ] ; then
-      echo "Running in non-interactive mode, assuming \"Yes\""
-      answer="Yes"
+      echo "Running in non-interactive mode, assuming \"No\""
+      answer="No"
     else
       read_timeout answer
     fi
@@ -143,7 +143,7 @@
         echo ""
         echo "*** You have to put pymolcas in any directory in your PATH"
         if [ "$INTERACTIVE" = "0" ] ; then
-          exit 3
+          exit 0
         fi
         break ;;
       * ) echo "Please answer yes or no"
